packages/fluentui/react-northstar/src/utils/index.ts

```diff
@@ -25,6 +25,7 @@ export { getKindProp } from './getKindProp';
 export * from './whatInput';
 
 export * from './commonPropInterfaces';
+export { screenReaderContainerStyles } from './accessibility/Styles/accessibilityStyles';
```

Comment:
@mnajdova @layershifter , is that OK to export this style to be able to use it in prototypes and docs?

Reply:
Should be ok as we don't depend on any site variables there.. Not sure if this is the right place though, but let's export it for now